### PR TITLE
Recreate PR #5

### PR DESCRIPTION
**Pull request by n0k0m3**
- visual-studio-code-bin.sh > /usr/bin/code launch script that allows launch options configurable from ~/.config/code-flags.conf
- Updated desktop entries exec to /usr/bin/code
- Updated PKGBUILD accordingly

Tested on 2 systems (Intel Iris Xe laptop, AMD Ryzen + NVIDIA desktop) with wayland options for electron, worked perfectly

Remade of #5 bc I closed and deleted the branch my mistake

### CODE REVIEW

File: visual-studio-code-bin/PKGBUILD [hunk 1]
```diff
@@ -62,6 +63,10 @@ package() {
   install -Dm 644 "${srcdir}/${_pkg}/resources/completions/zsh/_code" "${pkgdir}/usr/share/zsh/site-functions/_code"
 
   cp -r "${srcdir}/${_pkg}/"* "${pkgdir}/opt/${_pkgname}" -R
-  ln -s /opt/${_pkgname}/bin/code "${pkgdir}"/usr/bin/code
+
+  # Launcher
+  echo "  -> Edit ~/.config/code-flags.conf to add launch flags..."
```
Comment (dcelasun) [inline visual-studio-code-bin/PKGBUILD:68]: Don't do it like this. Instead, create an [.install file](https://wiki.archlinux.org/title/PKGBUILD#install) and do the echo in `post_install` and `post_upgrade`.

**Review by dcelasun — CHANGES_REQUESTED** [review]
Thanks for the PR! Just one change needed.

File: visual-studio-code-bin/PKGBUILD (hunk 1 above)
Comment (n0k0m3) [reply visual-studio-code-bin/PKGBUILD]: Done with 7298ec8

File: visual-studio-code-bin/PKGBUILD [hunk 2]
```diff
@@ -62,6 +64,9 @@ package() {
   install -Dm 644 "${srcdir}/${_pkg}/resources/completions/zsh/_code" "${pkgdir}/usr/share/zsh/site-functions/_code"
 
   cp -r "${srcdir}/${_pkg}/"* "${pkgdir}/opt/${_pkgname}" -R
-  ln -s /opt/${_pkgname}/bin/code "${pkgdir}"/usr/bin/code
+
+  # Launcher
+	install -m755 "${srcdir}/${_pkgname}-bin.sh" "${pkgdir}/usr/bin/code"
+  # ln -s /opt/${_pkgname}/bin/code "${pkgdir}"/usr/bin/code
```
Comment (dcelasun) [inline visual-studio-code-bin/PKGBUILD:70]: No need to comment out old code.
```suggestion
```

**Review by dcelasun — CHANGES_REQUESTED** [review]
Looks good! Please delete the commented line and bump `pkgrel` to 2.

After that, please squash your commits, rebase from master and change the commit message to:
```
[visual-studio-code-bin] Add wrapper for custom flags

This allows per-user configuration for native Wayland support.
```

File: visual-studio-code-bin/PKGBUILD (hunk 2 above)
Comment (n0k0m3) [reply visual-studio-code-bin/PKGBUILD]: Done

**Comment by dcelasun** [comment]
One last thing, you need to update `.SRCINFO` as well with `makepkg --printsrcinfo > .SRCINFO`.

**Comment by dcelasun** [comment]
Merged as [61589a0](https://github.com/dcelasun/pkgbuilds/commit/61589a01fc58cf7a96fae0c73310a326e443436e). Thanks for the PR!

**Pull request closed without merging**